docs: Notes about WAN Federation when using Vault as Connect CA (#11143)
hashicorp/consul: website/content/docs/connect/ca/vault.mdx

  exist, Consul will mount a new PKI secrets engine at the specified path with
  a [`max_lease_ttl`](https://www.vaultproject.io/api/system/mounts#max_lease_ttl)

>   of 8760 hours, or 1 year. This TTL value specifies the expiry period of the

Reviewer: Ya, this looks like a good clarification

>   root certificate and is currently not configurable. When enabling WAN Federation
>   over Mesh Gateways, each secondary datacenter must share the same `root_pki_path`
>   with the primary datacenter. 

Reviewer: ```suggestion
  root certificate and is currently not configurable. When WAN Federation
  is enabled, each secondary datacenter must use the same Vault cluster and share the same `root_pki_path`
  with the primary datacenter. 
```

>   with the primary datacenter. 

Reviewer: We've changed this now so that the TTL is actually configurable, so this part of the change conflicts with `main`.

- `IntermediatePKIPath` / `intermediate_pki_path` (`string: <required>`) -
  The path to a PKI secrets engine for the generated intermediate certificate.
  This certificate will be signed by the configured root PKI path. If this
  path does not exist, Consul will attempt to mount and configure this

>   automatically. When enabling WAN Federation over Mesh Gateways, each secondary 
>   datacenter must specify a unique `intermediate_pki_path`. 

Reviewer: ```suggestion
  automatically. When WAN Federation is enabled, every secondary 
  datacenter must specify a unique `intermediate_pki_path`. 
```
